ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/delete.md

```diff
@@ -6,6 +6,7 @@
 
 {% include [OLAP_not_allow_text](../../../../_includes/not_allow_for_olap_text.md) %}
 
+
```

Comment:
This extra space is not needed; one blank line should be enough. The same story below.

Reply:
Fix it